src/com/google/javascript/jscomp/ProcessCommonJSModules.java

```diff
@@ -140,6 +188,25 @@ public void visit(NodeTraversal t, Node n, Node parent) {
         visitRequireCall(t, n, parent);
       }
 
+      // Check for UMD pattern, pull out CommonJS branch and remove the rest.
+      // This way we end up with a "module.exports" or "exports" statement
+      // that can be reached.
+      if (n.isIf()) {
+        FindModuleExportsOrExports commonjsFinder = new FindModuleExportsOrExports();
+        NodeTraversal.traverse(compiler, n.getFirstChild(), commonjsFinder);
```

Comment:
pulling out `n.getFirstChild()` into a variable `condition` would make this and the code below more readable.